feat(dialog): add prop action (#581)
TuSimple/naive-ui: src/dialog/demos/enUS/index.demo-entry.md

| bordered | `boolean` | `false` | Whether to show border. |
| closable | `boolean` | `true` | Whether to show close icon. |
| content | `string \| (() => VNodeChild)` | `undefined` | Content, can be a render function. |

> | action | `() => VNodeChild` | `undefined` | Content of the operation area, needs to be a render function. |

Reviewer: ```suggestion
| action | `() => VNodeChild` | `undefined` | Content of the operation area, must be a render function. |
```
